### ISSUE TITLE
10.0.4 caused "[DEP0148] DeprecationWarning: Use of deprecated folder mapping..." warning

### ISSUE THREAD

**Issue by jhildenbiddle** [lo1tuma/eslint-plugin-mocha#323]
Updating dependencies for a project and noticed the following warning after updating to v10.0.4:

```shell
> eslint .

(node:91449) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./src/" in the "exports" field module resolution of the package at /Users/john/Dev/get-css-data/node_modules/ramda/package.json.
Update this package.json to use a subpath pattern like "./src/*".
(Use `node --trace-deprecation ...` to show where the warning was created)
```

After downgrading to v10.0.3 this warning no longer appears. Rather than post `package.json` content, here is a link to the project where I encountered the issue: https://github.com/jhildenbiddle/get-css-data

Thanks!

**Comment by lo1tuma** [comment]
Fixed in version 10.0.5 by https://github.com/lo1tuma/eslint-plugin-mocha/pull/326
